src/content/en/2021/performance.md

```diff
@@ -1,20 +1,532 @@
+  sheets_gid="730149797",
+  sql_file="tbt.sql"
+  )
+}}
+
+<p class="note"><strong>Note:</strong> The groups in the chart are based off of the Lighthouse score for TBT (e.g., >= 0.9 results in "good"). Due to rounding of the score, some TBT values slightly above 200ms get categorized as "good" (and similarly at the 600ms threshold).</p>
+
+Remember that the data is a single, throttled-CPU Lighthouse run through WebPageTest and does not reflect real user experiences. Yet, potential interactivity looked much worse when looking at TBT versus FID. The "real" evaluation of your interactivity is probably somewhere between. Thus, if your FID is "good", take a look at TBT in case you're missing some poor user experiences that FID can't catch yet. The same issues that cause poor FID also cause poor TBT.
+
+{{ figure_markup(
+  content="67 seconds",
+  caption="Longest TBT",
+  classes="big-number",
+  sheets_gid="1423728540",
+  sql_file="lcp_element_data.sql"
+  )
+}}
+
+## Conclusion
+
+We still have a long way to go to provide great user experiences, but we can take steps to improve them.
+
+First, you cannot improve performance unless you can measure it. A good first step here is to measure your site using real user devices and to set up real-user monitoring (RUM). You can get a flavor of how your site performs with Chrome users with the <a hreflang="en" href="https://rviscomi.github.io/crux-dash-launcher/">CrUX dashboard launcher</a> (if your site is in the dataset). You should set up a RUM solution that measures across multiple browsers. You can build this yourself or use one of many analytics vendors' solutions.
+
+Second, as new features in HTML, CSS, and JavaScript are released, make sure you understand them before implementing them. Use A/B testing to verify that adopting a new strategy results in improved performance. For example, don't lazy-load images above the fold. If you have a RUM tool implemented, you can better detect when your changes accidentally cause regressions.
+
+Additionally, continue to optimize for both FID (field/real-user data) and TBT (lab data). Take a look at the <a hreflang="en" href="https://web.dev/responsiveness/">proposal</a> for a new responsiveness metric and participate by providing feedback. A new <a hreflang="en" href="https://web.dev/smoothness/">animation smoothness metric</a> is also being proposed. In our quest for a faster web, change is inevitable and for the better. As we continue to optimize, you're participation is key.
+
+Finally, we saw that WordPress can impact the performance of the top 10M websites, and maybe more. This is a lesson that every CMS and framework should heed. The more we can set up smart defaults for performance at the framework level, the better we can make the web while also make developers' jobs easier.
```

Comment:
> Finally,

to

> Fourth,

or

> Finally, fourth,

Just to be consistent with the format you started.

> we saw that WordPress can impact the performance of the top 10M websites, and maybe more.

to

> we saw that WordPress impacted the performance of the top 10M websites (maybe more).

and

> The more we can set up smart defaults for performance at the framework level, the better we can make the web while also make developers' jobs easier.

to

> When we set smarter defaults at the framework level, the better the performance will be on the Web.

Reply:
The last rephrase removed the concept of making developer's jobs easier.

Reply:
I guess my feeling here is that I don't think that's your goal.  I think your goal is to make performance better and more accurate.  The by-product is the developer's job becoming easier, in this very narrow aspect.  I challenge the idea because I don't think you know if it would be easier (and what other complications this would mean if defaults are adopted).  I.e., would having defaults mean dev need to now build dashboards to view this in some way?  Will they have to pipeline it and store it and build a system to look at this?  Are additional monitoring steps needed by ops or dev to track this information (where before maybe it was on an individual basis and not a standard everyone follows)?  I think the developer job being easier is speculation.  I also, honestly, just couldn't fit it in and make it not sound like an extra by-product, structurally.  Probably because I was biased that it wasn't your end-game.

Reply:
No, developer experience is very important. Performance is too difficult which is why it is so bad. It's too much to learn combined with everything else on the front end. So the more we can do it at the framework level, the better it will be.

Reply:
OK.  Then it deserves its own sentence (if not, its own paragraph), not as an added thought at the end of the initial idea of improving the web experience for users.  For example:

> When we set smarter defaults on the framework level, the performance will improve on the Web.  This is because developers would no longer be individually solving the complex problem of capturing performance data.  Instead, they can focus on monitoring and responding to the consistent and correctly captured data provided by the framework.